flank-scripts/src/main/kotlin/flank/scripts/testartifacts/core/RemoveRemoteCopy.kt

```diff
@@ -0,0 +1,11 @@
+package flank.scripts.testartifacts.core
+
+import flank.scripts.github.getRelease
+
+fun Context.removeRemoteCopy() {
+    print("* Removing remote copy for $branch - ")
+    testArtifactsRepo().getRelease(branch)?.run {
+        delete()
+        println("OK")
```

Comment:
this is debbuging `OK` or development `OK`  😄  ?

Reply:
this is a standard output for this script.